### ISSUE TITLE
How can I use pdfium with anyhow?

### ISSUE THREAD

**Issue by kernelPanic0x** [ajrcarey/pdfium-render#58]
Hello.

I am trying to use anyhow to to use the '?' operator on multiple error types. But I get an ``` the trait bound `PdfiumError: StdError` is not satisfied ``` error. How can I resolve this?

```Rust
use pdfium_render::prelude::*;
use fltk::image::RgbImage;
use fltk::enums::ColorDepth;
use anyhow::Result;


pub fn load_pdf(path: &std::path::Path, page_width: u16, page_height: u16) -> Result<RgbImage> {
    
    // setup pdfium
    let pdfium = Pdfium::new(
        Pdfium::bind_to_library(Pdfium::pdfium_platform_library_name_at_path("./"))
            .or_else(|_| Pdfium::bind_to_system_library())?,
    );

    let doc = pdfium.load_pdf_from_file(&path, None)?;

    let render_config = PdfRenderConfig::new()
        .set_target_width(page_width)
        .set_maximum_height(page_height);

    let page = doc.pages().first()?;

    let rendered = page.render_with_config(&render_config)?;

    let img = rendered.as_image().to_rgb8();

    // convert rgb8 to fltk image object
    let fltk_img = RgbImage::new(&img.to_vec(), page_width as i32, page_height as i32, ColorDepth::Rgb8)?;

    Ok(fltk_img)
}

```

Do I have to implement a From-Trait? When so how do i do that?

**Comment by ajrcarey** [comment]
Hi @kernelPanic0x, thank you for reporting the issue. I'm happy to help.

PdfiumError is missing an implementation of `std::error::Error.` All errors handled by anyhow have to implement this trait. I have added an implementation in the latest commit. Set pdfium-render as a git dependency in your Cargo.toml to access it. You should then be able to build your example.

**Comment by ajrcarey** [comment]
Waiting on @kernelPanic0x for confirmation that the problem is resolved before merging into next release.

**Comment by ajrcarey** [comment]
As there have been no further comments, and I believe the issue is resolved, I am closing this issue. Feel free to re-open if you do not believe the problem has been satisfactorily resolved.

The change will be included in crate release 0.7.26.
